### DISCUSSION TITLE
Replacement for lh_SSL_SESSION_retrieve in latest openssl

### DISCUSSION THREAD

**Discussion by pb1234** [openssl/openssl#24151] [Q&A]
Hi , 

Our code is using openssl 1.0.2 fips32 version and using this api lh_SSL_SESSION_retrieve(s->sessions, c). We are upgrading to latest openssl but we dont find the definition of this API in fips32, what can be the replacement for this API in latest openSSL?

**Comment by nhorman** [comment]
The API call still exists.  Its defined via macro LHASH_OF(SSL_SESSION)

Though I don't believe its recommended that you access the session hash table outside the context of the ssl library
